web/src/compose_paste.ts

```diff
@@ -40,14 +40,74 @@ function image_to_zulip_markdown(
     return src ? "[" + title + "](" + src + ")" : (node.getAttribute("alt") ?? "");
 }
 
+// Returns 2 if there are two or more valid text nodes in the tree.
```

Comment:
Should be "Returns 2 or more if there are multiple valid text nodes in the tree".

The way the `+=` works, it can be 1 + 2 = 3.